front/pages/api/stripe/webhook.ts

```diff
@@ -399,9 +423,9 @@ async function handler(
  * Remove everybody except the most tenured admin
  * @param workspaceId
  */
-async function revokeUsersForDowngrade(workspaceId: number) {
+async function revokeUsersForDowngrade(auth: Authenticator) {
```

Comment:
Should we check auth role ?

Reply:
for a private function (i.e. use limited to webhook.ts, only role is readability), I would say it's not necessary, it's the role of the top level function to auth. IMHO here the function is just a cleaner, more readable way than inlining.
Glad to discuss

Reply:
There could be a point of adding `_` before the name to make it extra clear the fn is not to be exported

Reply:
Fair !